chat.py

```diff
@@ -111,22 +116,7 @@ def get_auto_db_path(latest: bool) -> str:
     return str(db_path)
 
 def get_auto_base_path() -> str:
```

Comment:
I know you've actually just done, what I've written before literally, but now I see `get_auto_base_path` is just not needed anymore, let's just directly use `get_cursor_workspace_path` in `discover`.